pyrasterframes/src/main/python/pyrasterframes/rasterfunctions.py

```diff
@@ -91,11 +91,11 @@ def rf_make_ones_tile(num_cols, num_rows, cell_type=CellType.float64()):
     return Column(jfcn(num_cols, num_rows, _parse_cell_type(cell_type)))
 
 
-def rf_rasterize(geometry_col, bounds_col, value_col, num_cols, num_rows):
+def rf_rasterize(geometry_col, bounds_col, value_col, num_cols_col, num_rows_col):
```

Comment:
need to update the function reference about this in particular 

> Parameters tile_columns and tile_rows are literals, not column expressions. The others are column expressions.

Reply:
Also interesting take a look at the little table that gets printed out as example there ate rf_rasterize. Perhaps we should just try to display the tile there instead (it all gets flattened to one line); or do away with the example?

Reply:
@vpipkt 

> need to update the function reference about this in particular
> 
> > Parameters tile_columns and tile_rows are literals, not column expressions. The others are column expressions.

Can you create a ticket for this?.... I'm actually not sure what you're referring to.